cmd/dcs-web/search/query-rewrite.go

```diff
@@ -14,31 +55,9 @@ func RewriteQuery(u url.URL) url.URL {
 	query := u.Query()
 
 	querystr := query.Get("q")
-	queryWords := []string{}
-	for _, word := range strings.Split(querystr, " ") {
-		fmt.Printf("word = %v\n", word)
```

Comment:
Is there a specific reason this `Printf` is here?  I had removed it, but can re-add it back into `rewriteFilters` if needed.